install_requirements.sh

```diff
@@ -59,7 +60,7 @@ done
 # NOTE: If a newly-fetched version of the executorch repo changes the value of
 # NIGHTLY_VERSION, you should re-run this script to install the necessary
 # package versions.
-NIGHTLY_VERSION=dev20240507
+NIGHTLY_VERSION=dev20240508
```

Comment:
why do you need to update this file?

Reply:
The dev20240507 version has a dependency issue with torchvision for aarch64 unfortunately.

Reply:
You also have to update `.ci/docker/ci_commit_pins/pytorch.txt`

Reply:
Ok, got it.

Reply:
Not sure how the hash corresponds to the dev version. Assumed it was the git sha-1 for the dev build. 
**dev20240507**: 
```
python -c 'import torch; print(torch.version.git_version)'
e4f522910a192a919111277ce3e1c0fb0c8482bd
```
but the .ci/docker/ci_commit_pins/pytorch.txt contains 48b6c8dbc376db4406a979b35cd6909bcb428931
so this is something different?

Can I find it on https://hud.pytorch.org/ or where do I go look it up?

Reply:
https://hud.pytorch.org/hud/pytorch/pytorch/nightly

For example, if click on "2024-05-07 nightly release", it will show https://hud.pytorch.org/pytorch/pytorch/commit/e4f522910a192a919111277ce3e1c0fb0c8482bd

which contains 48b6c8dbc376db4406a979b35cd6909bcb428931

There are some failures for 2024-05-08 and 05-09 versions. Maybe update to "2024-05-10" nightly release? (https://hud.pytorch.org/pytorch/pytorch/commit/8d3ad715c85fd3c2a87445f6ef3ead75f7d9c9ab)

Reply:
Thanks Mergen!
I'll update the commit and verify that 2024-05-10 works.

Reply:
Works fine!